stable/ignite/values.yaml

```diff
@@ -0,0 +1,51 @@
+# Default values for ignite.
+replicaCount: 2
+
+image:
+  repository: apacheignite/ignite
+  tag: 2.6.0
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+rbac: yes
+
+ignite:
```

Comment:
Is there a reason for this to be nested? Are there other components?

Reply:
Everything under `ignite:` is application configurations. Everything top-level is release configurations. We followed this logic. Do you think it's better to put everything under `ignite also` to top-level?

Reply:
It caught me a bit off guard since the chart/app and image already reference ignite. Additionally its what ive seen most commonly:
- Multi component apps nest things
- Single component apps have them all top-level

Reply:
I see, I've moved everything to the top level.